testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/injection/resource/resourceref/jboss-web.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jboss-web xmlns="http://www.jboss.com/xml/ns/javaee"
+<jboss-web xmlns="urn:jboss:jakartaee:1.0"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xsi:schemaLocation="http://www.jboss.com/xml/ns/javaee http://www.jboss.org/j2ee/schema/jboss-web_7_2.xsd"
-           version="7.2">
+           xsi:schemaLocation="urn:jboss:jakartaee:1.0 https://www.jboss.org/schema/jbossas/jboss-web_15_0.xsd"
```

Comment:
should we not use a location for `urn:jboss:jakartaee:1.0`? The url https://www.jboss.org/schema/jbossas/jboss-web_15_0.xsd does not exist, and probably will not.